docs/README.md

```diff
@@ -1,4 +1,4 @@
-<br />
+local_development.md<br />
```

Comment:
not sure if it intentional but looking weird I have tried to remove it https://github.com/astronomer/ask-astro/pull/121

Reply:
Thank you